pkg/plugin/trivy/plugin.go

```diff
@@ -478,6 +495,21 @@ func (s *scanner) getPodSpecForClientServerMode(spec corev1.PodSpec, credentials
 			})
 		}
 
+		ref, err := name.ParseReference(container.Image)
```

Comment:
The code for setting the TRIVY_INSECURE env is duplicated in two places: for Standalone and ClientServer mode. Can we instead refactor this logic to a private method and invoke it twice?

Reply:
added a private method